### ISSUE TITLE
Accessibility - WCAG 2.4.6 Headings and Labels, label is missing in the search-box 

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#144]
Label is missing in the search-box (all example pages and all components; description can be invisible)
![image](https://cloud.githubusercontent.com/assets/7510842/3226849/b32b7fc6-f073-11e3-97e8-54d7c9438cc6.png)

Label is missing in the following element (example pages)
![image](https://cloud.githubusercontent.com/assets/7510842/3226926/eeeab38c-f074-11e3-9935-ce88101666f1.png)

This was not ok in the original version
